angular-rails-app/app/assets/javascripts/property/property.html

```diff
@@ -1,8 +1,7 @@
 <section style="min-height: 65px;margin-top:105px;">
   <div class="container">
     <p id="breadcrumb">
-      <a class="breadcrumb_top" ui-sref="home">Home /</a> Building /
-      {{vm.data.street_address | titleCase}}, {{vm.data.city}} {{vm.data.state}}
+      <a class="breadcrumb_top" ui-sref="home">Home /</a> Building 
```

Comment:
Ah, I was saying that the address is just below, so there's really no need for it in the breadcrumbs. 😄